src/modules/emote_menu/components/EmoteMenu.jsx

```diff
@@ -100,30 +100,32 @@ export default function EmoteMenu({toggleWhisper, appendToChat, onSetTip}) {
         selected={selected}
       />
       <Divider className={styles.divider} />
-      <div className={styles.content}>
+      <div className={styles.contentContainer}>
         <Sidebar
           className={styles.sidebar}
           section={section}
-          categories={emoteMenuViewStore.getCategories()}
-          onChange={(eventKey) => setSection({eventKey, scrollTo: true})}
-        />
-        <Emotes
-          className={styles.emotes}
-          search={search}
-          section={section}
-          onClick={handleClick}
-          setKeyPressCallback={setKeyPressCallback}
-          rows={emoteMenuViewStore.rows}
-          setSelected={setSelected}
-          onSection={(eventKey) => setSection({eventKey, scrollTo: false})}
+          onClick={(eventKey) => setSection({eventKey, scrollTo: true})}
+          categories={{
+            top: emoteMenuViewStore.getCategories(CategoryPositions.TOP),
+            middle: emoteMenuViewStore.getCategories(CategoryPositions.MIDDLE),
+            bottom: emoteMenuViewStore.getCategories(CategoryPositions.BOTTOM),
+          }}
         />
+        <div className={styles.content}>
+          <Emotes
+            className={styles.emotes}
+            search={search}
+            section={section}
+            onClick={handleClick}
+            setKeyPressCallback={setKeyPressCallback}
+            rows={emoteMenuViewStore.rows}
+            setSelected={setSelected}
+            onSection={(eventKey) => setSection({eventKey, scrollTo: false})}
+          />
+          <Divider className={styles.divider} />
+          <Preview emote={selected} isFavorite={selected == null ? false : emoteMenuViewStore.hasFavorite(selected)} />
```

Comment:
nit: invert this for readability: `selected != null ? emoteMenuViewStore.hasFavorite(selected) : false`